### DISCUSSION TITLE
Pointers for session

### DISCUSSION THREAD

**Discussion by DesignrKnight** [dscnitrourkela/workshop-adv-git#1] [General]
Let us use this to keep track of topics we will cover and any topics if anyone has any suggestions

**Comment by Thesmader** [comment]
Please make a notion page on DSC Notion and give edit access.

**Reply by Thesmader** [reply]
https://www.notion.so/Advanced-Git-and-GitHub-workshop-cb54fd2f739a4effa4e377c32d105803?v=cc0aeb9c60594ac3a3a6603dad1c6cbc
Created!
